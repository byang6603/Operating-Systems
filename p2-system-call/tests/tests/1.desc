Sanity Test: Launch test program from shell and print hello world. 
