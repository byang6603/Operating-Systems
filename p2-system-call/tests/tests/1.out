XV6_TEST_OUTPUT Hello world!
XV6_TEST_OUTPUT Crash returned -1.
XV6_TEST_OUTPUT Crash returned -1.
